### PR TITLE
Issue #13345: EnableFileLengthCheckExamplesTest

### PR DESCRIPTION
**Pull request by Coding-Aliens**
Issue #13345: EnableFileLengthCheckExamplesTest
mvn clean verify passed.
There is little to do in the .txt file, please review if there's anything left.

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example2.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/*xml
+<module name="Checker">
+  <module name="FileLength">
+    <property name="max" value="1500"/>
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example2.java:4]: Please do us a favor.
Change max to be "5" and make Example2 to be violation.

Please make example3 to show usage of `fileExtensions` property. In example3 set fileExtensions=txt
And show fact that java is skipped and violation is on txt file only .

Here is example of how we did file extension example in another Check https://github.com/checkstyle/checkstyle/commit/6c2405453f800bdb742c7e31411b57cc826a0d00#diff-e5ba5fe56a9f8ea2a93c39f4df1ddc2b1359c3828695652f98a01893b418b200R140

https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/2cbdc32_2024163715/checks/misc/newlineatendoffile.html#Example4-code

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example1.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <module name="FileLength"/>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.filelength;
+
+public class Example1 {
+  // xdoc section -- start
+  // xdoc section -- end
+}
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example1.java:12]: Please put such comments outside of class and please add placing java code to template to show that no violation is on small class

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example2.java (hunk 1 above)
Comment (Coding-Aliens) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example2.java]: Done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example1.java (hunk 2 above)
Comment (Coding-Aliens) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example1.java:14]: Done

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
There are only configs, please add to xdoc files java and txt files.
After each config, there should be context of target example file.
Default limit is 2000, but any file with less lines is good, so showing some java code after default config is also ok .

Please see how examples are layouted https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/2cbdc32_2024163715/checks/misc/newlineatendoffile.html#Example4-code

**Comment by Coding-Aliens** [comment]
Updated

**Comment by romani** [comment]
GitHub, generate website

**Comment by romani** [comment]
GitHub, generate website

**Review by romani — APPROVED** [review]
ok to merge if CI pass

**Pull request merged**